js-intro.js

```diff
@@ -7,37 +7,48 @@ var mantra = "Be the dev"
 
 // 1a. Write the code that determines if there is a 'B' in mantra.
 
-
+console.log(mantra.includes("B"))
 
 // 1b. Write the code that determines if there is an 'x' in mantra.
 
-
+console.log(mantra.includes("x"))
 
 // 1c. Write the code that determines if there is a 'v' in mantra.
 
-
+console.log(mantra.includes("v"))
 
 // 1d. STRETCH: Write a function that can take two arguments - a string and a letter and returns a sentence telling the user if the letter exists within the string.
+ 
+const myName = (letter) => {
+    if (letter.includes('Michael')){
+        return letter + " exists within the string.";
+    } else{
+        return letter + " does not exist within the string.";
+    }
+};
 
-
+console.log(myName('M'));
 
```

Comment:
Really close. If you add another argument for the string your code will be more dynamic:
const myName = (string, letter) => {
    if(string.includes(letter)){
        return letter + " exists within the string.";
    } else{
        return letter + " does not exist within the string.";
    }
};
console.log(myName('Michael','M'));